modules/apps/document-library-google-docs/document-library-google-docs/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-	compileInclude group: "com.liferay", name: "org.freemarker", version: "2.3.29.LIFERAY-PATCHED-5"
```

Comment:
this commit should come after this https://github.com/rafaprax/liferay-portal/pull/1419/commits/5c0d804b8af46fb8b3cadf615b7ec22168cdf293#diff-8eecd0fb1ae096fcfd2a3868810c20ada706bb28f33c280a049445a13fc47a2fR28 change and the message should be something like "Auto SF"